tests/vmi_iothreads_test.go

```diff
@@ -68,8 +68,8 @@ var _ = Describe("[sig-compute]IOThreads", decorators.SigCompute, func() {
 		})
 
 		It("[test_id:4122]Should honor shared ioThreadsPolicy for single disk", func() {
-			policy := v1.IOThreadsPolicyShared
-			vmi.Spec.Domain.IOThreadsPolicy = &policy
+			addPolicy := libvmi.WithIOThreadsPolicy(v1.IOThreadsPolicyShared)
+			addPolicy(vmi)
```

Comment:
Sorry, this looks like a hack to me. The builders are not aimed to be used like this and while

I prefer you drop the shared `vmi` and place them in the tests directly to do this.

As an exception this may make sense, but here it is used as a pattern and it looks bad IMO.